[2024-08-22T14:43:12.638+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Credit_score_pipeline_v02.train_model scheduled__2024-08-21T00:00:00+00:00 [queued]>
[2024-08-22T14:43:12.648+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Credit_score_pipeline_v02.train_model scheduled__2024-08-21T00:00:00+00:00 [queued]>
[2024-08-22T14:43:12.649+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-08-22T14:43:12.666+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): train_model> on 2024-08-21 00:00:00+00:00
[2024-08-22T14:43:12.693+0000] {standard_task_runner.py:60} INFO - Started process 298 to run task
[2024-08-22T14:43:12.696+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Credit_score_pipeline_v02', 'train_model', 'scheduled__2024-08-21T00:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/credit_score.py', '--cfg-path', '/tmp/tmp17xl4f2c']
[2024-08-22T14:43:12.699+0000] {standard_task_runner.py:88} INFO - Job 266: Subtask train_model
[2024-08-22T14:43:12.774+0000] {task_command.py:423} INFO - Running <TaskInstance: Credit_score_pipeline_v02.train_model scheduled__2024-08-21T00:00:00+00:00 [running]> on host 056d689e0f27
[2024-08-22T14:43:12.861+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Credit_score_pipeline_v02' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-21T00:00:00+00:00'
[2024-08-22T14:43:14.424+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-08-22T14:43:14.434+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Credit_score_pipeline_v02, task_id=train_model, execution_date=20240821T000000, start_date=20240822T144312, end_date=20240822T144314
[2024-08-22T14:43:14.482+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-22T14:43:14.502+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
